### ISSUE TITLE
Check for Names in JourneyDetail

### ISSUE THREAD

**Issue by highsource** [patrickbr/db-api-to-gtfs#2]
In some cases `JourneyDetail` produces an error. This can be caught by checking

```
    if 'JourneyDetail' in data and 'Names' in data['JourneyDetail']:
```

instead of simply

```
    if 'JourneyDetail' in data:
```

**Comment by patrickbr** [comment]
Makes sense! Fixed in 709abc60bd922d3a1b1b49ee3cd902900db03ccb.

**Comment by highsource** [comment]
Many thanks. :+1:
